src/pages/login.js

```diff
@@ -27,9 +27,18 @@ const Login = ({ firebase }) => {
   return (
     <>
       <SEO title="Login" route="/login" />
-      <Centered>
-        <Logo size="medium" />
-        <h1>Authenticating...</h1>
+      <Centered style={{ paddingTop: "5%", paddingBottom: "5%" }}>
```

Comment:
This page and logout page are using the exact same markup but with different text, they can be abstracted into a single component